invenio_vocabularies/config.py

```diff
@@ -162,6 +163,9 @@
 }
 """Vocabulary type search configuration."""
 
+SUBJECTS_EUROSCIVOC_FILE_URL = "https://op.europa.eu/o/opportal-service/euvoc-download-handler?cellarURI=http%3A%2F%2Fpublications.europa.eu%2Fresource%2Fdistribution%2Feuroscivoc%2F20231115-0%2Frdf%2Fskos_ap_eu%2FEuroSciVoc-skos-ap-eu.rdf&fileName=EuroSciVoc-skos-ap-eu.rdf"
```

Comment:
We have this issue in general with all the contrib-style vocabularies. Ideally these config values should be loaded only if the contrib is used, but I don't see any other issue besides the config namespace pollution.

Alternative is that we break them into separate packages or extras, but this would complicate integrating this by a lot.